.github/pull_request_template.md

```diff
@@ -11,20 +11,13 @@
 ## Post-release Monitoring Plan <sub><sup>(what's your plan to monitor the changes to make sure they have the intended effect)</sup></sub>
 
 ## Checklist
-- [ ] Update PR title to follow Conventional Commits/Semantic Versioning
-  - For more information read [What should my PR Title be](https://github.com/clearbanc/code/tree/staging/packages#what-should-my-pr-title-be)
 - [ ] Add a brief description of the purpose of this PR.
 - [ ] Add screenshots for this PR.
 - [ ] Add testing instructions for this PR.
 - [ ] Introduce or modify relevant documentation
-- [ ] Apply appropriate label (e.g. `needs review`).
-- [ ] Remove changes that aren't related to the purpose of this PR.
-- [ ] Remove debugging code that was accidentally committed.
 - [ ] Ensure unhappy paths are considered and tested
-- [ ] Ensure there is a plan for monitoring post deployment
-- [ ] Ensure the build is passing.
-- [ ] Test your changes in the deployed PR preview environment / [run migration locally](https://github.com/clearbanc/code/tree/staging/packages/migrations#run-migrations).
-- [ ] Leave pre-emptive comments on changes you'd like to discuss.
+- [ ] Ensure there is a plan for monitoring post-deployment
```

Comment:
- [ ] Ensure unhappy paths are considered/tested, and there is a plan for monitoring post-deployment

Reply:
consolidate the two WCGW and monitoring